contracts/utils/ChainportLERC20.sol

```diff
@@ -1,308 +1,1375 @@
+     * @param   beneficiary is an address of token sender
+     */
+    function withdrawTokenIfStuck(address token, address beneficiary) external onlyChainportCongress {
+        uint256 amount = IERC20(token).balanceOf(address(this));
+        IERC20(token).transfer(beneficiary, amount);
+        emit StuckTokenWithdrawn(token, beneficiary, amount);
+    }
+
+    /// Lossless Compliance
+
+    event ConfirmBlacklisted(address blacklisted);
+
+    function lssTransfer(address recipient, uint256 amount) internal {
+        if (isLosslessOn) {
+            lossless.beforeTransfer(_msgSender(), recipient, amount);
         }
-        emit Transfer(address(0), account, amount);
     }
 
-    function _approve(address owner, address spender, uint256 amount) internal virtual {
-        _allowances[owner][spender] = amount;
-        emit Approval(owner, spender, amount);
+    function lssTransferFrom(address sender, address recipient, uint256 amount) internal {
+        if (isLosslessOn) {
+            lossless.beforeTransferFrom(_msgSender(), sender, recipient, amount);
+        }
     }
 
-    function _burn(address account, uint256 amount) internal virtual {
-        require(account != address(0), "LERC20: burn from the zero address");
+    modifier lssBurn(address account, uint256 amount) {
+        if (isLosslessOn) {
+            lossless.beforeBurn(account, amount);
+        } 
+        _;
+    }
 
-        uint256 accountBalance = _balances[account];
-        require(accountBalance >= amount, "LERC20: burn amount exceeds balance");
-        unchecked {
-            _balances[account] = accountBalance - amount;
+    modifier lssMint(address account, uint256 amount) {
+        if (isLosslessOn) {
+            lossless.beforeMint(account, amount);
+        } 
+        _;
+    }
+
+    function setLosslessController(address _controller) public onlyClassifiedAuthority {
+        require(_controller != address(0), 
+        "BridgeMintableToken: Controller cannot be zero address.");
+        require(_controller != address(lossless), 
+        "BridgeMintableToken: Cannot set same address.");
+
+        lossless = ILssController(_controller);
+    }
+
+    function setLosslessAdmin(address newAdmin) external onlyClassifiedAuthority {
+        require(newAdmin != admin, "LERC20: Cannot set same address");
+        admin = newAdmin;
+    }
+
+    function transferOutBlacklistedFunds(address[] calldata from) external {
+        require(_msgSender() == address(lossless), "LERC20: Only lossless contract");
+
+        uint256 fromLength = from.length;
+        uint256 totalAmount = 0;
+        
+        for(uint256 i = 0; i < fromLength;i++) {
+            address fromAddress = from[i];
+            require(confirmedBlacklist[fromAddress], "LERC20: Blacklist not confirmed");
+            uint256 fromBalance = super.balanceOf(fromAddress);
+            super.subBalance(fromAddress, fromBalance);
```

Comment:
`super` not needed?

Reply:
Removed on commit [22b8de3](https://github.com/Lossless-Cash/lossless-v3/pull/105/commits/22b8de34cd7a6c77e6169d07ceba8ffd23cd5e95)